azureDatabaseUtils/src/main/java/bio/terra/workspace/azureDatabaseUtils/create/CreateDatabaseDao.java

```diff
@@ -62,12 +62,32 @@ else pgaadauth_create_principal_with_oid(:roleName, :userOID, 'service', false,
         .get(0);
   }
 
+  public boolean createRole(String roleName) {
```

Comment:
Can you add documentation (in particular, about what the return value signifies)?